lease/lessor_test.go

```diff
@@ -213,16 +214,19 @@ func TestLessorRenew(t *testing.T) {
 // TestLessorRenewExtendPileup ensures Lessor extends leases on promotion if too many
 // expire at the same time.
 func TestLessorRenewExtendPileup(t *testing.T) {
-	oldRevokeRate := leaseRevokeRate
-	defer func() { leaseRevokeRate = oldRevokeRate }()
-	leaseRevokeRate = 10
```

Comment:
The race happens from the `runLoop` that keeps reading 'leaseRevokeRate`. And according to the stack trace, `TestLessorGrant ` was running concurrently.